rflx/pyrflx/typevalue.py

```diff
@@ -663,7 +670,19 @@ def set(self, fld: str, value: Union[bytes, int, str, Sequence[TypeValue], Bitst
 
         if all([self.__simplified(o.condition) == FALSE for o in self._model.outgoing(Field(fld))]):
             self._fields[fld].typeval.clear()
-            raise ValueError("value does not fulfill field condition")
+            value_repr = None
+            if isinstance(value, Bitstring):
+                value_repr = str(value)[0:20]
+                value_repr += f"(length: {len(str(value))})"
```

Comment:
Why not `len(value)` instead of `len(str(value))`?

Reply:
IMHO the current representation does not make clear that the value is truncated:
```
ValueError: value 01100110011001100011(length: 144) for field Payload  [...]
```

Why not just showing the full bitstring? It is probably quite long, but I think that is not a big issue.

There is also a blank space missing between value and length.

Reply:
Fixed by 829c5c0